### PR TITLE
venv

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

**Comment by Ilya-Krylov** [comment]
Ilya-Krylov-patch-2

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

File: README.md [hunk 1]
```diff
@@ -25,20 +25,20 @@ inference.
...
 
 4. Install `ote_cli` package:
     ```
     pip3 install -e ote_cli/
     ```
-    
+
 5. Instantiate templates and create virtual environments:
    ```
    python3 tools/instantiate.py --destination model_templates --verbose --init-venv
```
Comment (druzhkov-paul) [inline README.md:44]: Kinda off-topic, but is it still relevant?

**Review by druzhkov-paul — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Ilya-Krylov) [reply README.md:44]: Will fix in next pr

**Pull request merged**